projects/ion/src/lib/use-table/use-table.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core';
+import { IonApiService } from './ApiService.service';
+import { BnTable } from '../../public-api';
+
+interface User {
+  id: string;
+  nome: string;
+  data_criacao: string;
+}
+
+@Component({
+  selector: 'ion-use-table',
+  templateUrl: './use-table.component.html',
+  styleUrls: ['./use-table.component.scss'],
+})
+export class IonUseTableComponent extends BnTable<User> {
+  constructor(service: IonApiService) {
+    super({
+      service,
+      tableConfig: {
+        columns: [{ label: 'Nome', sort: true, key: 'nome' }],
```

Comment:
Will you ever need to have a name column?

Reply:
> Will you ever need to have a name column?

This component its for tests on storybook.